config/config.go

```
@@ -0,0 +1,29 @@
+package config
+
+import (
+    "github.com/spf13/viper"
+)
+
+type Config struct {
+    SERVER_ADDRESS  string `mapstructure:"SERVER_ADDRESS"`
+    CSV_SOURCE      string `mapstructure:"CSV_SOURCE"`
+}
+
+var PokedexConfig = LoadConfig()
+
+/* Reads configuration from pokemon.env file */
+func LoadConfig() (config Config) {
```

This function should return an error if there's something that might cause a failure.

This is a key point, as if something goes wrong while reading the configuration, something worse could happen when trying to run the project with a bad configuration.